experimental/PropertyDDS/packages/property-proxy/src/errors.ts

```diff
@@ -0,0 +1,66 @@
+/*!
```

Comment:
I wonder why some renamings are tracked in the git view as a rename from js to ts with some changes in the file, while others are considered a remove of the js file with an addition of a new ts files were renamed the same way and changes were done in a separate commit.